src/oc_bifrost/rebar.lock

```diff
@@ -23,40 +23,41 @@
   {git,"git://github.com/chef/epgsql-1.git",
        {ref,"34b4182f0e21f9189ddd7b2e290f01a9e7d93bf1"}},
   1},
+ {<<"fs">>,{pkg,<<"fs">>,<<"6.1.1">>},1},
  {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
  {<<"jiffy">>,
-  {git,"git://github.com/davisp/jiffy.git",
+  {git,"https://github.com/davisp/jiffy.git",
        {ref,"051a74338c089c39f09532188bc82cf1adedbdc8"}},
   0},
  {<<"lager">>,
   {git,"git://github.com/erlang-lager/lager.git",
-       {ref,"f26a8ea0f4c2ed6506a54e0b9890edd8148b9500"}},
+       {ref,"91575d4479c897451edf138c10fa86c3125ec52e"}},
   0},
  {<<"meck">>,
   {git,"git://github.com/eproxus/meck.git",
-       {ref,"9753464ac1ebc0fc1a6777322bc871eaed82be35"}},
+       {ref,"06192a984750070ace33b60a492ca27ec9bc6806"}},
   1},
  {<<"mixer">>,
   {git,"git://github.com/chef/mixer.git",
        {ref,"58ded93d5c47675899d8e5e1589270f340ea66c5"}},
   0},
  {<<"mochiweb">>,
   {git,"git://github.com/mochi/mochiweb.git",
-       {ref,"cedc22f66a8e3f3885fb06babc0c0582418508f8"}},
+       {ref,"070594e4d66163d662ac7e3bfb75dadcc922dd7c"}},
   0},
  {<<"observer_cli">>,
   {git,"git://github.com/zhongwencool/observer_cli.git",
-       {ref,"bcc0dfaa2013a436ce7e597731acab6577321b43"}},
+       {ref,"baa70569bccc5508e9839e20768540ef3cdca016"}},
   0},
  {<<"opscoderl_wm">>,
   {git,"git://github.com/chef/opscoderl_wm.git",
        {ref,"7b222296b2f0eecd86f3c858f2185d21d55e8fa4"}},
   0},
  {<<"pooler">>,
-  {git,"git://github.com/chef/pooler.git",
-       {ref,"681c355abaacc5487ddf41a84b9ed53151a765fe"}},
+  {git,"git://github.com/seth/pooler.git",
```

Comment:
This must be getting pulled in from an upstream lib, but `seth/pooler` is not up to date with the latest updates we've made to `chef/pooler`.

Reply:
Thanks for catching that.  It appears it was being pulled in by sqerl:
```
./rebar3 tree
├─ sqerl─1.0.0 (git repo)
│  ├─ envy─0.5.0+build.44.ref0148fb4 (git repo)
│  ├─ epgsql─3.1.0 (git repo)
│  └─ pooler─1.5.2+build.205.ref9c28fb4 (git repo)
```